forge-gui/res/cardsfolder/upcoming/ozolith_the_shattered_spire.txt

```diff
@@ -0,0 +1,11 @@
+Name:Ozolith, the Shattered Spire
+ManaCost:1 G
+Types:Legendary Artifact
+R:Event$ AddCounter | ActiveZones$ Battlefield | ValidCard$ Creature.YouCtrl+inZoneBattlefield,Artifact.YouCtrl+inZoneBattlefield | ValidCounterType$ P1P1 | ReplaceWith$ AddOneMoreCounters | Description$ If one or more +1/+1 counters would be put on an artifact or creature you control, that many plus one +1/+1 counters are put on it instead.
```

Comment:
Are the `inZoneBattlefield` necessary with the ActiveZones ?

Reply:
That only defined where it works from but it could still affect other zones

Not sure if it was needed for +1/+1 too but Skullbriar is sometimes weird